### DISCUSSION TITLE
Drag & drop column reordering in admin UI

### DISCUSSION THREAD

**Discussion by hades200082** [payloadcms/payload#2075] [Feature Requests & Ideas]
On a collection listing page when adding columns to the table the newly added column is added to the left of the table.

This makes for poor UX in LTR-based locales because it means you then have to remove all columns and re-add them all in the reverse order to that which you want them displayed.

`defaultColumns` only partially solves this (as it means editors are less likely to need to change the displayed columns). 

It would be great if you could have an option to reorder columns, either by a simple drag/drop or by toggling a reorder mode and using arrows or something.

**Comment by jacobsfletch** [comment]
@hades200082 check out this PR! https://github.com/payloadcms/payload/pull/1865

**Reply by hades200082** [reply]
Seen. When will that make it into a release?

**Reply by jacobsfletch** [reply]
It's in the final stages of testing and should be out this week.
